addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7387,7 +7396,7 @@ msgstr ""
 
 #: xbmc/windows/GUIWindowFileManager.cpp
 msgctxt "#16206"
-msgid "Failed to delete at least one file"
+msgid "Failed to delete at least one file.[CR]See log for more details."
```

Comment:
I always wonders about this, why cant the specific log entry be pulled into the message box Ill never know.

However I agree or the rewording of the later and will make them all consitent, in fact I was already on them :)